mw-webapp/src/component/table/columns.tsx

```diff
@@ -22,17 +22,24 @@ const getObjectArrayItem = (arrayItem: JobDone | PlanForNextPeriod | CurrentProb
   );
 };
 
-const getStringArrayItem = (arrayItem: string, index: string) => {
+
+/**
+ * Render a string item within a div element.
+ *
+ * @param {object} params - The parameters for rendering the string item.
+ * @param {string} params.text - Text to be rendered
+ * @param {string} params.index - The key for the rendered div element, to ensure React elements have unique keys.
+ * @param {boolean} [params.isDone] - Optional. If true, the item is styled as completed.
+ *
+ * @returns {JSX.Element} The rendered string item.
+ */
+const renderStringItem = ({text, index, isDone}: {text:string; index:string; isDone?: boolean}): JSX.Element => {
   return (
-    (!arrayItem) ?
-      <div />
-      :
-    //TODO: task #65 use flag instead of first index
-      <div key={index}>
-        <div className={arrayItem[INDEX_OF_CHECK_MARK] === "✓" ? styles.completed : styles.notCompleted}>
-          {arrayItem}
-        </div>
+    <div key={index}>
```

Comment:
index must not be used as key

Reply:
Renamed parameter, in fact it's not a index